app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.map do |movie|
+      movie_params(movie)
+    end
+    render json: @movies
+  end
+
+  def show
+    movie = Movie.find(params[:id])
+    @transformed_movie = movie_params(movie)
+    render json: @transformed_movie
+  end
+
+  def create
+    Movie.create(movie_params)
+  end
+
+  def update
+    movie = Movie.find(params[:id])
+    movie.update(movie_params)
+  end
+
+  def delete
+    movie = Movie.find(params[:id])
+    movie.destroy
+  end
+
+  private
+
+  def movie_params(movie)
```

Comment:
This name for that method is misleading in my opinion if you want to build a JSON response. You could use `serialize_movie` for example.